dataactbroker/fileRoutes.py

```diff
@@ -376,6 +387,11 @@ def certify_submission(submission):
         if submission.publish_status_id == PUBLISH_STATUS_DICT['published']:
             return JsonResponse.error(ValueError("Submission has already been certified"), StatusCode.CLIENT_ERROR)
 
+        window = get_gtas_window()
+        if window is None:
```

Comment:
Same as above, can this just be `if not window:`? If it can't that's fine, but it seems cleaner